api/src/main/resources/cb_search_api.yaml

```diff
@@ -296,56 +296,88 @@ definitions:
       - hasAttribute
     properties:
       id:
-        description: id of the criteria
+        description: >
+          Primary identifier which is unique within a CDR version. Value may
+          not be stable across different CDR versions.
         type: integer
         format: int64
       parentId:
-        description: parent id of the criteria
+        description: >
+          The parent id of the criteria. 0 if this is the root node of a
+          criteria tree.
         type: integer
         format: int64
       type:
-        description: type of criteria
+        description: The tree type of this criteria, see TreeType.
+        # TODO: Use TreeType enum?
         type: string
       subtype:
-        description: subtype of this criteria
+        description: The tree subtype of this criteria, see TreeSubType.
+        # TODO: Use TreeSubType enum?
         type: string
       code:
+        # TODO: Some of the criteria table has null codes, but it's required
+        # here. What does that mean and how does that manifest in the API?
```

Comment:
Code can be removed from required. The api doesn't depend on the code being set unless it's ICD9/ICD10 type. We have validation here: https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/cohortbuilder/querybuilder/CodesQueryBuilder.java#L121

Reply:
Done